media-libs/vulkan-layers/vulkan-layers-1.3.239.ebuild

```diff
@@ -37,8 +37,6 @@ DEPEND="${RDEPEND}
 	)
 "
 
-PATCHES="${FILESDIR}/${P}-Build-shared-libs.patch"
```

Comment:
we can't do this for multiple reasons
1) this version is in stable, we can't just modify it

2) we always version bump for changes

Reply:
So I need to create a 1.3.239-r2.ebuild tagged as ~amd64, right?

Reply:
yeah so leave .1.3.239 and 1.3.239 untouched and make a -r2 with the changes and makes sure its all in unstable, thanks